README.md

```diff
@@ -6,6 +6,13 @@
        src="https://scan.coverity.com/projects/12056/badge.svg"/>
 </a>
 
+## Getting Started
+Launch `Terminal` in Ubuntu 16.04
```

Comment:
See "Pre-Requisites" below. It lists the generic dependencies for any Linux distribution. And below that there are the example commands for one of those distros, Ubuntu.  Keep things related to Ubuntu there.